src/backend/distributed/relay/relay_event_utility.c

```diff
@@ -536,6 +547,70 @@ AppendShardIdToConstraintName(AlterTableCmd *command, uint64 shardId)
 }
 
 
+static bool
+ExtendRowReferencesWalker(Node *node, RowRefWalkerState *state)
+{
+	if (node == NULL)
```

Comment:
Sure, that's fine.  I was just trying to follow the example described here:  https://github.com/postgres/postgres/blob/master/src/backend/nodes/nodeFuncs.c#L1772

If Citus' style is to (typically) have one return point, that's fine.